app/src/App.tsx

```diff
@@ -0,0 +1,16 @@
+import { Route, Routes } from 'react-router-dom';
+import { Navbar } from './components/Navbar';
+import { routes } from './routes/routes';
+
+export const App = () => {
```

Comment:
Makes sense why you added a react app for testing. I don't know much about rollup, but I assume it packages only what is in `src/` so this `app/` directory will be ignored

Reply:
Yeah that's right.